src/box/iproto.cc

```diff
@@ -661,6 +753,22 @@ iproto_connection_try_to_start_destroy(struct iproto_connection *con)
 static inline void
 iproto_connection_close(struct iproto_connection *con)
 {
+	struct remote_txn *remote_txn = con->txn;
+	if (remote_txn != NULL &&
+	    remote_txn->txn != NULL &&
+	    remote_txn->disconnect != NULL) {
+		struct cmsg *msg = (struct cmsg *) remote_txn->disconnect;
+		remote_txn->disconnect = NULL;
```

Comment:
Here we have a memory leak i think. When net_finish_rollback_on_disconnect called and then remote_txn_destroy, we have remote_txn->disconnect  == NULL and memory is not freed